handbook/product/design/design-process.md

```diff
@@ -32,27 +32,26 @@ While asynchronous communication is a core attribute of remote work, key moments
       - As our users are developers, Sourcegraph developers make great interview subjects. However, we need to acknowledge that they do have biases about features and this research must be augmented
   - Interview external users
       - Where the project requires and time permits, conduct research with these users to inform your designs
-  - Placeholder:
-      - Additional research methods
   - Customer requests and support tickets
       - Our users are passionate and provide excellent feedback. Each project should utilize our collections of this feedback in [Productboard](https://sourcegraph.productboard.com/)
   - Analogous designs
   - Competitive designs
-  - Findings publishing and review
-      - Findings should be recorded in google docs and announced in the corresponding team channel or #product on Slack
-          - Research documents should contain a clear summary of findings and charts of supporting data are helpful.
-      - If the size of the project requires, a review meeting should be held to discuss key findings.
+  - Usability tests
+    - When updating existing product features, first conduct usability tests to inform redesign decisions
   - Process
       - A GitHub issue should be created to track the work
       - Research is conducted according to requirements
-      - Publish research to Google Drive
+      - Findings publishing and review
+        - Findings should be detailed in a google doc and announced in the corresponding team channel or #product on Slack
+        - Findings should contain a clear summary, links to resources and supporting data
+        - Conduct a syncronous review meeting with key stakeholders based on side of project or importance of findings
       - Announce research in Slack
```

Comment:
@rrhyne Good changes! I have only one suggestion here. It seems like announcing research in Slack is already mentioned in line 45. Maybe we can remove this line (48)? :)

Reply:
Done!